.lune/util/generator/generator.luau

```diff
@@ -0,0 +1,51 @@
+local fs = require("@lune/fs")
+local net = require("@lune/net")
+
+local Generator = {}
+
+function Generator.removeLegacyFiles()
+	if fs.isDir("docs/classes") then
+		fs.removeDir("docs/classes")
+	end
+
+	fs.writeDir("docs/classes")
+end
```

Comment:
I'm not sure what this is supposed to be doing here.

Reply:
`docs` being the folder where you throw the MD files.
`classes` being a child folder, where you throw the generated MD files from moonwave

it's written like this, because in the event you want to write a `guide`, you can:

`docs/guides/abc.md`

Reply:
There's not much point to doing this. Please make it a flat directory structure like I mentioned.